runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -1224,20 +1679,14 @@ TR::Register *J9::Power::TreeEvaluator::awrtbariEvaluator(TR::Node *node, TR::Co
    return NULL;
    }
 
-TR::Register *J9::Power::TreeEvaluator::irdbarEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+TR::Register *iConcurrentScavengeSequence(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Since we're renaming this function already, is it possible to rename to `TR::TreeEvaluator::generateSoftwareReadBarrier` so as to stay consistent with the Z codegen and make the name more meaningful?

[1] https://github.com/eclipse/openj9/blob/fde4c8087c1e700ea6d396fa269b4f9aa4d375bb/runtime/compiler/z/codegen/J9TreeEvaluator.cpp#L4171